opendkim/opendkim.conf.5.in

```diff
@@ -179,6 +179,11 @@ requires superuser access. A warning will be generated if
 .I UserID
 is not also set.
 
+.TP
+.I CheckSigningTable (Boolean)
+If set to yes, it walks the SigningTable on boot when it loads the config
```

Comment:
As I commented other place,  loading config (and the opportunity of the check) may occure other than on boot, by receiving SIGHUP signal.

Reply:
https://github.com/trusteddomainproject/OpenDKIM/pull/228/commits/ac1d2c6503109460a9c5ab9b07efc7942b82ac2e